### PR TITLE
SQS broker updates

### PR DESCRIPTION
**Pull request by alexbilbie**
This PR has two changes:

1. Removes unused session variable
2. Adds a new broker option to set an instantiated SQS client which will be used instead of creating a new client with shared config state

### CODE REVIEW

**Comment by alexbilbie** [comment]
The TravisCI check is failing due to unrelated tests:

```
--- FAIL: TestGeocode (0.17s)
	geocode_test.go:23: returned status: OVER_QUERY_LIMIT
FAIL
FAIL	github.com/micro/go-plugins/micro/bot/command/geocode	0.172s
?   	github.com/micro/go-plugins/micro/bot/command/whereareyou	[no test files]
?   	github.com/micro/go-plugins/micro/bot/input/discord	[no test files]
?   	github.com/micro/go-plugins/micro/bot/input/telegram	[no test files]
?   	github.com/micro/go-plugins/micro/disable_rpc	[no test files]
?   	github.com/micro/go-plugins/micro/gzip	[no test files]
?   	github.com/micro/go-plugins/micro/index	[no test files]
?   	github.com/micro/go-plugins/micro/ip_whitelist	[no test files]
?   	github.com/micro/go-plugins/micro/metadata	[no test files]
```

File: broker/sqs/options.go [hunk 1]
```diff
@@ -67,3 +69,13 @@ func WaitTimeSeconds(seconds int64) broker.SubscribeOption {
 		o.Context = context.WithValue(o.Context, waitTimeSecondsKey{}, seconds)
 	}
 }
+
+// SetSQSClient receives an instantiated instance of an SQS client which is used instead of initialising a new client
+func SetSQSClient(svc *sqs.SQS) broker.Option {
```
Comment (asim) [inline broker/sqs/options.go]: Call this SQSClient

File: broker/sqs/options.go [hunk 2]
```diff
@@ -3,8 +3,10 @@ package sqs
 import (
 	"github.com/micro/go-micro/broker"
 	"golang.org/x/net/context"
+	"github.com/aws/aws-sdk-go/service/sqs"
 )
 
+type sqsClient struct {}
```
Comment (asim) [inline broker/sqs/options.go]: Call this sqsClientKey

**Comment by alexbilbie** [comment]
@asim I've pushed the changes you've requested

File: broker/sqs/options.go [hunk 3]
```diff
@@ -71,11 +71,11 @@ func WaitTimeSeconds(seconds int64) broker.SubscribeOption {
 }
 
 // SetSQSClient receives an instantiated instance of an SQS client which is used instead of initialising a new client
-func SetSQSClient(svc *sqs.SQS) broker.Option {
+func SetSQSClient(SQSClient *sqs.SQS) broker.Option {
```
Comment (asim) [inline broker/sqs/options.go]: The function name should be SQSClient, not the argument.

Comment (asim) [reply broker/sqs/options.go]: Or even just Client since that would then be `sqs.Client`

**Pull request merged**